projects/core/src/occ/adapters/product/occ-product.adapter.ts

```diff
@@ -26,6 +27,36 @@ export class OccProductAdapter implements ProductAdapter {
     protected requestsOptimizer: OccRequestsOptimizerService
   ) {}
 
+  loadRealTimeStock(productCode: string): Observable<string> {
+    return this.http.get(this.getEndpoint(productCode, ProductScope.UNIT)).pipe(
```

Comment:
MAJOR: can we move this api call to get unit into a separate method so that it can be reused if we need to fetch unit in any other scenario?

Reply:
done